react/__tests__/pickInstallments.test.ts

```diff
@@ -40,3 +43,68 @@ describe('pickInstallments', () => {
     expect(pickedInstallments[index].NumberOfInstallments).toBe(15)
   })
 })
+
+describe('pickMaxInstallmentsOption', () => {
```

Comment:
I think would be great we have a test that uses both settings, the payment system, and installmentQuantity